.circleci/workflows.yml

```diff
@@ -1342,18 +1342,7 @@ jobs:
     resource_class: << parameters.resource_class >>
     steps:
       - update_known_hosts
-      - when:
-          condition:
-            equal: [ *windows-executor, << parameters.executor >> ]
-          steps: # issue with ssh keys on windows
-            - checkout
-      - unless:
-          condition:
-            equal: [ *windows-executor, << parameters.executor >> ]
-          steps:
-            - run:
-                name: checkout
-                command: git clone -b "$CIRCLE_BRANCH" "$CIRCLE_REPOSITORY_URL" . --depth 1
+      - checkout
```

Comment:
Is this slower now that we aren't doing `--depth 1`?

Reply:
Why not just leave it then?

Reply:
Issues with 3rd-party contributors & the release-readiness step due to the git-history not being present. I want to take a clean look at it without it impact the release.